app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_user!
+
+  def index
+    @users = policy_scope(User)
+    authorize User
+    @roles = Role.all
+    @current_user = current_user
+  end
+
+  def update
+    @user = User.find(params[:id])
+    authorize @user
+    @user.update(secure_params)
+    render users_url
+  end
+
+  def destroy
+    @users = User.all
+    @users.find(params[:id]).destroy
```

Comment:
just saw the same in https://github.com/rokor/Ruby_Combats/pull/24/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R7 

please read the comment

Reply:
I've made rebase from the branch, you are linked to, because I use the same template and controller. Because of that I have 2 commits in this PR and action 'destroy' from that branch.